css/styles.css

```diff
@@ -3,135 +3,132 @@
-    font-size: 40px;
-    color: #172B4D;
-    font-family: Poppins;
-    font-weight: 700;
-    border:0px;
-    line-height: 52px;
+.primary-text {
+  margin-top: 114px;
+  margin-left: 24px;
+  text-align: left;
+  height: 150px;
+  width: 327px;
+  font-size: 40px;
+  color: #172b4d;
+  font-family: Poppins, Times, serif;
+  font-weight: 700;
+  border: 0;
+  line-height: 52px;
 }
 
 .secondary-text {
-    margin-top: 12px;
-    margin-left: 24px;
-    text-align:left;
-    height: 146px;
-    width: 327px;
-    font-size: 16px;
-    color: #172B4D;
-    font-family: Poppins;
-    font-style: normal;
-    font-weight: 400;
-    border:0px;
-    line-height: 24px;
+  margin-top: 12px;
+  margin-left: 24px;
+  text-align: left;
+  height: 146px;
+  width: 327px;
+  font-size: 16px;
+  color: #172b4d;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  border: 0;
+  line-height: 24px;
 }
 
 .connect-text {
-    margin-top: 12px;
-    margin-left: 24px;
-    height: 24px;
-    width: 120px;
-    font-size: 16px;
-    color: #7F8CFF;
-    font-family: Poppins;
-    font-style: normal;
-    font-weight: 500;
-    border:0px;
-    line-height: 24px;
+  margin-top: 12px;
+  margin-left: 24px;
+  height: 24px;
+  width: 120px;
+  font-size: 16px;
+  color: #7f8cff;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  border: 0;
+  line-height: 24px;
 }
 
 .socialmedia-icons {
-    margin-top: 12px;
-    margin-left: 24px;
-    width: 170.17px;
-    height: 20px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content:space-between ;
-  }
-
-  .twitter-icon {
-    background-image: url("../images/twitter-icon.png");
-    width: 19.99px;
-    height: 19.96px;   
-  }
+  margin-top: 12px;
+  margin-left: 24px;
+  width: 170.17px;
+  height: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
 
-  .linkedin-icon {
-    background-image: url("../images/linkedin-icon.png");
-    width: 20px;
-    height: 20px; 
-  }
+.twitter-icon {
+  background-image: url("../images/twitter-icon.png");
+  width: 19.99px;
+  height: 19.96px;
+}
 
-  .m-icon {
-    background-image: url("../images/m-icon.png");
-    width: 20px;
-    height: 20px; 
-  }
+.linkedin-icon {
+  background-image: url("../images/linkedin-icon.png");
+  width: 20px;
+  height: 20px;
+}
 
-  .cat-icon {
-    background-image: url("../images/cat-icon.png");
-    width: 20px;
-    height: 20px; 
-  }
+.m-icon {
+  background-image: url("../images/m-icon.png");
+  width: 20px;
+  height: 20px;
+}
 
-  .fingers-icon {
-    background-image: url("../images/fingers-icon.png");
-    width: 20px;
-    height: 20px; 
-  }
+.cat-icon {
+  background-image: url("../images/cat-icon.png");
+  width: 20px;
+  height: 20px;
+}
 
+.fingers-icon {
+  background-image: url("../images/fingers-icon.png");
+  width: 20px;
+  height: 20px;
+}
```

Comment:
- [ ] Please make your page responsive by using percentages(%) for your widths. Currently, your page has a fixed width because you used pixels(px). Please fix that 👌